packages/test-utils/src/reporter.ts

```diff
@@ -187,6 +188,28 @@ export class ReporterError extends Error {
   }
 }
 
+function getFallbackRunTitle() {
+  // for CI runs we don't want to set an explicit title
+  // dashboard is meant to use the commit/PR information retrieved on CI
+  if (process.env.CI) {
+    return;
+  }
+
+  let gitChild;
+
+  try {
+    gitChild = spawnSync("git", ["rev-parse", "--abbrev-ref", "HEAD"]);
```

Comment:
I'm not super fond of doing this in this package. I started adding this to the source metadata but it also didn't feel right. I concluded this isn't really part of what we call source here. That shape can be rechecked here:
https://github.com/replayio/replay-cli/blob/cdb96e8259d27ec3792b5e05478232b985b3f3b1/packages/replay/src/metadata/source.ts#L235-329

Also, since this is configurable using `config.runTitle` it also didn't fit well into how `sourceMetadata` gets initialized. We also had this `this.runTitle` computed synchronously when the source metadata is computed asynchronously